Formula/vim.rb

```diff
@@ -55,6 +56,7 @@ def install
     # Homebrew will use the first suitable Perl & Ruby in your PATH if you
     # build from source. Please don't attempt to hardcode either.
     system "./configure", "--prefix=#{HOMEBREW_PREFIX}",
+                          "--with-local-dir=#{HOMEBREW_PREFIX}",
```

Comment:
```suggestion
```
I'm not convinced this is needed, so I'd rather not add this here.

Reply:
Ok that's fine. I still think it's likely some linker flags in the environment is allowing it to build successfully somehow, but I don't really care about this issue too much tbh. I'll remove it.

Reply:
`brew` sanitises the build environment, so any linker flags available on the build machine should be available when you build from source locally.